presto-product-tests/conf/presto/etc/file_auth

```diff
@@ -0,0 +1,2 @@
+userPBKDF2:1000:5b4240333032306164:f38d165fce8ce42f59d366139ef5d9e1ca1247f0e06e503ee1a611dd9ec40876bb5edb8409f5abe5504aab6628e70cfb3d3a18e99d70357d295002c3d0a308a0
+userBCrypt:$2y$10$BqTb8hScP5DfcpmHo5PeyugxHz5Ky/qf3wrpD7SNm8sWuA3VlGqsa
```

Comment:
how about having this file under `presto-product-tests/conf/docker/fileauth/presto-master/etc/presto/file_auth` and the using docker volume in `presto-product-tests/conf/docker/fileauth/docker-compose.yml`?